mage/template.go

```diff
@@ -108,3 +108,48 @@ func list() error {
 }
 
 `
+
+var mageTpl = `// +build mage
+
+package main
+
+import (
+	"os/exec"
+	"fmt"
+	"github.com/magefile/mage/mg" // mg contains helpful utility functions, like Deps
+)
+
+// Default target to run when none is specified
```

Comment:
Specify here also that if no default is set, running just `mage` will list the available targets.